README.md

```diff
@@ -170,13 +170,13 @@ To build v2.0 documentation:
 
 1. Initialize the `2.0` branch in your local devdocs repository.
 
-    ```shell
+    ```bash
```

Comment:
What is the difference between: `bash`, `shell`, and `terminal`?

Reply:
Shell and Terminal do not include the bash prompt `$`.

<img width="1181" alt="Screen Shot 2019-09-20 at 9 28 17 AM" src="https://user-images.githubusercontent.com/13662379/65334931-0a38ae00-db89-11e9-8f4f-267ae91d24d9.png">

Reply:
👍

Reply:
Sorry, to be clear, that's the difference in the files we render to HTML. In the README, i don't think it makes a difference.

Reply:
````
```bash
echo 'Hello, world!'
```

```shell
echo 'Hello, world!'
```

```terminal
echo
```
````

<img width="298" alt="Screen Shot 2019-09-20 at 09 32 52" src="https://user-images.githubusercontent.com/12731225/65335245-ab276900-db89-11e9-9028-cff953cb3d1b.png">

Reply:
Maybe it's common knowledge when to use each of these, but I'll need to brush up on when to use which---`bash`, `terminal`, `shell`.

Reply:
Use `bash` for terminal commands when you want the `$` symbol to appear at the beginning.
Use `shell` for terminal commands when you don't need the `$` symbol.
Use `terminal` when you want to demonstrate terminal output.

````
To teach your machine some courtesy, run in your terminal:

```bash
echo 'Good day to you, Shrie!'
```

Enjoy a polite respond:

```terminal
Good day to you, Shrie!
```
````